docs/source/configuration/slots.md

```diff
@@ -0,0 +1,89 @@
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from Plone ClassicUI artifact `Viewlets`.
```

Comment:
I think this is the correct markup to use Intersphinx. We should check the Netlify preview build to verify it links to Plone 6 docs correctly.

```suggestion
This concept is inherited from the Plone Classic UI {doc}`plone:classic-ui/viewlets`.
```